packages/features/calAIPhone/interfaces/AIPhoneService.interface.ts

```diff
@@ -301,6 +301,22 @@ export interface AIPhoneServiceProvider<T extends AIPhoneServiceProviderType = A
     message: string;
   }>;
 
+  /**
+   * Create a new inbound agent
+   */
+  createInboundAgent(params: {
+    name?: string;
+    phoneNumber: string;
+    userId: number;
+    teamId?: number;
+    workflowStepId: number;
+    userTimeZone: string;
```

Comment:
These types seem duplicate with _createAgent's_ types. Let's abstract out the type and reuse.